appveyor.yml

```diff
@@ -0,0 +1,20 @@
+version: "{build}"
+os: Windows Server 2012 R2
+branches:
+  only:
+    - master
+test: off
+deploy: off
+
+install:
+  - appveyor DownloadFile https://atom.io/download/windows -FileName AtomSetup.exe
+  - AtomSetup.exe /silent
+
+build_script:
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - SET PATH=%LOCALAPPDATA%\atom\bin;%PATH%
+  - apm clean
+  - apm install
+  # TODO: add specs (#13)
+  - npm tst
```

Comment:
Same thing, I'd put it as `npm test` myself.

I guess it doesn't really matter though.